runner/file.go

```diff
@@ -184,14 +184,14 @@ func writeSheet(rows [][]string, w *jsonutil.StreamEncoder) error {
 	var header []string
 	isHeader := true
 
-	row := map[string]any{}
 	for _, r := range rows {
 		if isHeader {
 			header = r
 			isHeader = false
 			continue
 		}
 
+		row := map[string]any{}
```

Comment:
There is a huge perf impact to this. Rather than doing this we should set the empty cells to nil each time.

Reply:
I'm not sure if allocating an empty map has more or less impact than iterating over `header` and setting map entries to nil.

Therefore, feel free to go for the way you prefer. 😉